amical/mf_pipeline/bispect.py

```diff
@@ -988,8 +988,20 @@ def _add_infos_header(infos, hdr, mf, pa, filename, maskname, npix):
     infos["filename"] = filename
     infos["maskname"] = maskname
     infos["isz"] = npix
-    if "SPHERE" not in infos.instrument:
-        infos["hdr"] = hdr
+
+    # HACK: astropy _HeaderCommentaryCards are registered as mappings,
+    # so munch tries to access their keys, leading to attribute error
+    # to prevent this, we remove commentary cards as a temporary fix.
+    # See:
+    # https://github.com/SydneyAstrophotonicInstrumentationLab/AMICAL/issues/31
+    # https://github.com/astropy/astropy/issues/11866
```

Comment:
great job documenting the hack ! can you also mention what's the current latest version number in Astropy ?

Reply:
I added the version and the date below the HACK comment.

Reply:
Perfect, thanks !